libraries/AP_OpenDroneID/AP_OpenDroneID.cpp

```diff
@@ -255,7 +257,19 @@ void AP_OpenDroneID::send_location_message()
     if (!ahrs.get_location(current_location)) {
         return;
     }
-    const uint8_t uav_status = hal.util->get_soft_armed()? MAV_ODID_STATUS_AIRBORNE : MAV_ODID_STATUS_GROUND;
+    uint8_t uav_status = hal.util->get_soft_armed()? MAV_ODID_STATUS_AIRBORNE : MAV_ODID_STATUS_GROUND;
+#if HAL_PARACHUTE_ENABLED
+    // set emergency status if chute is released
+    const auto *parachute = AP::parachute();
+    if (parachute != nullptr && parachute->released()) {
```

Comment:
If parachute is nullptr, would you constitute that as enough for a `MAV_ODID_STATUS_REMOTE_ID_SYSTEM_FAILURE` status?

Reply:
no, it just means parachute is not enabled on this vehicle

Reply:
@tridge Do you see a need to also check the parachute enabled flag? `parachute->enabled()`